WordPress/src/main/java/org/wordpress/android/ui/mysite/QuickStartRepository.kt

```diff
@@ -93,7 +95,8 @@ class QuickStartRepository
             refresh()
         }
         val quickStartTaskTypes = refresh.mapAsync(coroutineScope) {
-            dynamicCardStore.getCards(siteId).dynamicCardTypes.map { it.toQuickStartTaskType() }.onEach { taskType ->
+            val quickStartTaskTypes = getQuickStartTaskTypes(siteId)
+            quickStartTaskTypes.onEach { taskType ->
```

Comment:
Minor (🔍): Maybe the best equivalent to the previous piece of login is this below instead:
```
getQuickStartTaskTypes(siteId).onEach { taskType ->
    ...
}
```

Reply:
In 16fd588